konfera/models/organizer.py

```diff
@@ -0,0 +1,24 @@
+from konfera.models.countries import COUNTRIES
+from django.db import models
+from konfera.models.abstract import KonferaModel
+
+
+class Organizer(KonferaModel):
```

Comment:
I have not realized fields are required unless defined otherwise. I will change it. The question is whether the about_us field should not be required.